style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: large;
```

Comment:
I think we should use numeric value instead of using large keyword as it will affect the default in your website to make it responsive. 
```suggestion
  font-size: 12px;
```